payment-server

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env bash
```

Comment:
the file rename wasn't picked up as such by the git. It's the file removed & file created instead.
 Not a big deal tho, just noticed